src/add-nav-options-icon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import singleSpaReact from "single-spa-react";
+import { navigateToUrl } from "single-spa";
+import AppSwitcher20 from "@carbon/icons-react/lib/app-switcher/20";
+
+export default singleSpaReact({
```

Comment:
What is this?

Reply:
This is the app-nav icon from carbon icons

Reply:
How is `export default singleSpaReact({` the app-nav icon from carbon icons? Don't you confuse line 5 (app switcher icon) with line 7 (usage of Single SPA React for unknown reasons, which is the line that is covered by my comment)?

Reply:
Sir the add-nav-options-icon.tax it's actually an extension implementation that I did. I don't really know much about the singleSpaReact. I saw it like a routine when writing extensions in other files that's why I wrote it that way. But I saw that the extension was not too important so I removed it.